libparsec/crates/types/src/local_manifest.rs

```diff
@@ -108,11 +108,23 @@ impl Chunk {
     pub fn copy_between_start_and_stop(
         &self,
         chunk_data: &[u8],
+        offset: u64,
```

Comment:
maybe rename to `offset_after_start` given having a `offset` param in a function called `copy_between_start_and_stop` is a bit puzzling...

Reply:
See this comment: https://github.com/Scille/parsec-cloud/pull/6577#discussion_r1510899522